crates/hir_def/src/resolver.rs

```diff
@@ -649,6 +656,12 @@ impl HasResolver for TypeAliasId {
     }
 }
 
+impl HasResolver for ConstParamId {
```

Comment:
not sure this impl makes sense / is necessary? a const param isn't a definition that introduces a new scope anywhere